src/Symfony/Component/Messenger/Command/FailedMessagesRetryCommand.php

```diff
@@ -177,14 +177,14 @@ private function runWorker(string $failureTransportName, ReceiverInterface $rece
 
             $this->displaySingleMessage($envelope, $io);
 
-            $shouldHandle = $shouldForce || $io->confirm('Do you want to retry (yes) or delete this message (no)?');
+            $shouldHandle = $shouldForce || $io->confirm('Do you want to retry this message?');
 
             if ($shouldHandle) {
                 return;
             }
 
-            $messageReceivedEvent->shouldHandle(false);
-            $receiver->reject($envelope);
+            //$messageReceivedEvent->shouldHandle(false);
+            //$receiver->reject($envelope);
```

Comment:
I don't know this code, but the change seems incorrect. The function does nothing after `$shouldHandle` is set.

Reply:
> I don't know this code

yes me neither, those comments was to tests mainly
but it seems that if removed, the command loops over and over on same failed message if you answer no